andela/AndelaEatsUI
# #163643908 Fix bugs and improve the UI

#### What does this PR do?

- Fix bugs and improve the UI

#### Description of Task to be completed?

- implemented all the toast message when an action fails
- improved the UI to make follow the Andela conventions

#### How should this be manually tested?

- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `bg-fix-bugs-163643908`
- run these commands `npm run build` and `npm run dev` 
- Log in as a user and navigate through the app.  You should now see changes made to the UI.

#### What are the relevant pivotal tracker stories?
- [#163643908](https://www.pivotaltracker.com/story/show/163643908)

#### Background Context

#### Screenshots (If Applicable)
